<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Poojas-MacBook-Pro.local" ignored="0" name="TestFailure" tests="1" failures="1" timestamp="2022-09-06T21:19:24 PDT" time="0.006" errors="0">
  <testcase name="DoAuth" time="0.003" classname="regressiontest.TestFailure">
    <failure type="java.lang.AssertionError" message="Auth failed">
      <![CDATA[java.lang.AssertionError: Auth failed
at regressiontest.TestFailure.DoAuth(TestFailure.java:11)
at java.util.ArrayList.forEach(ArrayList.java:1259)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- DoAuth -->
</testsuite> <!-- TestFailure -->
